core/lib/db_connection/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "zksync_db_connection"
+version = "0.1.0"
+edition = "2021"
+authors = ["The Matter Labs Team <hello@matterlabs.dev>"]
+homepage = "https://zksync.io/"
+repository = "https://github.com/matter-labs/zksync-era"
+license = "MIT OR Apache-2.0"
+keywords = ["blockchain", "zksync"]
+categories = ["cryptography"]
+
+[dependencies]
+zksync_health_check = { path = "../health_check" }
+zksync_types = { path = "../types" }
+async-trait = "0.1"
+
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+sqlx = { version = "0.7.3", default-features = false, features = [
```

Comment:
To discuss: maybe it makes sense to re-export sqlx from this crate, so dal crates reuse it?

Reply:
Generally if a crate assumes that it must be used with some other crate in combination, it's often a good idea to re-export said crate.

This way the dependent crates:
1. Would have only one dependency instead of 2.
2. Would always get a correct version of the "other" crate and wouldn't have to guess which version should it use.